client/plots/matrix.serieses.js

```diff
@@ -72,7 +72,7 @@ export function getSerieses(data) {
 			for (const [i, value] of values.entries()) {
 				const cell = Object.assign({ key, siblingCells }, cellTemplate)
 				cell.valueIndex = i
-				let cellProps = setCellProps[t.tw.term.type]
+				const cellProps = t.grp.type == 'hierCluster' ? setCellProps['hierCluster'] : setCellProps[t.tw.term.type]
```

Comment:
geneExpression is not defined in `setCellProps{}`

Reply:
@congyu-lu please address this, so adding a gene exp term via Variables button can work.

Reply:
it is added now